src/NuGet.Clients/PackageManagement.VisualStudio/ProjectSystems/BuildIntegratedProjectSystem.cs

```diff
@@ -18,7 +20,7 @@ namespace NuGet.PackageManagement.VisualStudio
     /// <summary>
     /// A nuget aware project system containing a .json file instead of a packages.config file
     /// </summary>
-    public class BuildIntegratedProjectSystem : BuildIntegratedNuGetProject
+    public class BuildIntegratedProjectSystem : ProjectJsonBuildIntegratedNuGetProject
```

Comment:
>BuildIntegratedProjectSystem [](start = 17, length = 28)

Needs a new name.  ProjectJsonBuildIntegratedProjectSystem

would VS in the name make more sense?